test/component/CustomsDeclarationStatusSpec.scala

```diff
@@ -49,16 +52,16 @@ class CustomsDeclarationStatusSpec extends ComponentTestSpec with AuditService w
 
   private val apiSubscriptionKeyForXClientIdV3 = apiSubscriptionKeyForXClientIdV2.copy(version = VersionThree)
 
-  private val validResponse =
-    """<stat:declarationManagementInformationResponse xmlns:stat="http://gov.uk/customs/declarations/status-request">
+  private def validResponse(acceptanceDateVal : String) =
+    raw"""<stat:declarationManagementInformationResponse xmlns:stat="http://gov.uk/customs/declarations/status-request">
```

Comment:
`raw` typo

Reply:
The raw is needed as otherwise the argument is not evaluated and ends up as just '$acceptanceDateVal'